src/indra_cogex/client/queries.py

```diff
@@ -787,8 +800,14 @@ def get_evidences_for_stmt_hash(
                RETURN n.evidence"""
         % stmt_hash
     )
+
+    # Add limit and offset
+    if offset > 0:
```

Comment:
if you add an offset/limit you also need to add an `ORDER BY` to ensure the queries are consistent

Reply:
Fixed in https://github.com/bgyori/indra_cogex/pull/88/commits/416ae8cdb4514232ab914d31d4000d173509722b